km/km_filesys.c

```diff
@@ -1898,7 +1900,7 @@ static inline size_t fs_core_write_nonsocket(char* buf, size_t length, km_file_t
       fnote->data = file->ofd;   // default to ofd. override based on file type
       // If a non-std{in,out,err} pipe contains data we don't snapshot
       if (fd > 2 && ioctlfionread(fd) > 0) {
-         km_errx(1, "Can't take a snapshot, fifo fd %d has buffered data", fd);
+         km_warn("Can't take a snapshot, fifo fd %d has buffered data", fd);
```

Comment:
We probably don't want this change.
It is needed to get the node snapshot to work.